crossplane/README.md

```diff
@@ -0,0 +1,146 @@
+# Crossplane on Management Clusters
+
+Managed Crossplane offering is under development as per customers' requests.
+This document discusses all the required elements of Crossplane's deployment
+and the design decisions we make along the way.
+
+## Crossplane building blocks
+
+For ease of describing each proposal, we will divide a fully functional
+Crossplane deployment into three stages.
+
+### Stage 1 - Crossplane
+
+The first stage is Crossplane itself. This is not enough to manage any
+infrastructure yet.
+
+- Crossplane Operator
+- Crossplane RBAC Manager Operator
+- Custom Resource Definitions:
+  - `compositeresourcedefinitions.apiextensions.crossplane.io`
+  - `compositionrevisions.apiextensions.crossplane.io`
+  - `compositions.apiextensions.crossplane.io`
+  - `configurationrevisions.pkg.crossplane.io`
+  - `configurations.pkg.crossplane.io`
+  - `controllerconfigs.pkg.crossplane.io`
+  - `locks.pkg.crossplane.io`
+  - `providerrevisions.pkg.crossplane.io`
+  - `providers.pkg.crossplane.io`
+  - `storeconfigs.secrets.crossplane.io`
+- Other Kubernetes resources
+
+### Stage 2 - Crossplane Providers
+
+Infrastructure is managed by Providers. Creating a
+`providers.pkg.crossplane.io` resource will cause Crossplane operators to
```

Comment:
From what I have seen the providers can be installed as a configuration as well, for example in the official docs:

- https://crossplane.io/docs/v1.9/getting-started/install-configure.html#install-configuration-package

but the AWS Jet Provider is installed as:

- https://github.com/crossplane-contrib/provider-jet-aws#getting-started

Reply:
Using CLI or supplying a Helm chart value does the same - it creates a `Provider` CR. Am I missing something?

Reply:
Just thinking that there are multiplel ways to install packages, hence the `install configuration` and `install provider` commands for `kubectl crossplane`. From what I can tell, with `configuration` you can set constraints and multiple providers e.g: https://crossplane.io/docs/v1.9/getting-started/create-configuration.html#build-and-push-the-configuration compared to a raw provider: https://github.com/crossplane-contrib/provider-aws/blob/master/package/crossplane.yaml

Reply:
I think it works by creating a Provider object no matter which way you request it.